website/www/site/data/en/documentation_runners.yaml

```diff
@@ -0,0 +1,34 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: { text: "DirectRunner:", link: /documentation/runners/direct/ }
+  icon: icons/documentation/runners/beam-icon.svg
+  description: Runs locally on your machine – great for developing, testing, and debugging.
+- name: { text: "FlinkRunner:", link: /documentation/runners/flink/ }
+  icon: icons/documentation/runners/flink-icon.svg
+  description: Runs on <a href="https://flink.apache.org">Apache Flink</a>.
+- name: { text: "SparkRunner:", link: /documentation/runners/spark/ }
+  icon: icons/documentation/runners/spark-icon.svg
+  description: Runs on <a href="https://spark.apache.org">Apache Spark</a>.
+- name: { text: "DataflowRunner:", link: /documentation/runners/dataflow/ }
+  icon: icons/documentation/runners/dataflow-icon.svg
+  description: Runs on <a href="https://cloud.google.com/dataflow">Google Cloud Dataflow</a>, a fully managed service within Google Cloud Platform.
+- name: { text: "SamzaRunner:", link: /documentation/runners/samza/ }
+  icon: icons/documentation/runners/samza-icon.svg
+  description: Runs on <a href="https://samza.apache.org">Apache Samza</a>.
+- name: { text: "NemoRunner:", link: /documentation/runners/nemo/ }
+  icon: icons/documentation/runners/nemo-icon.svg
+  description: Runs on <a href="https://nemo.apache.org">Apache Nemo</a>.
+- name: { text: "JetRunner:", link: /documentation/runners/jet/ }
+  description: Runs on <a href="https://jet.hazelcast.org/">Hazelcast Jet</a>.
+- name: { text: "Twister2Runner:", link: /documentation/runners/twister2/ }
+  description: Runs on <a href="https://twister2.org/">Twister2</a>.
```

Comment:
I wonder if external links should open in new tabs?

Reply:
yea, you're right, I fixed this